e2e/testdata/fn-render/entrypoint-success/.expected/config.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+stdErr: |
+  [RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1.3"
+  [PASS] "gcr.io/kpt-fn/set-namespace:v0.1.3" in 0s
+  [RUNNING] "gcr.io/kpt-fn/set-labels:v0.1.4[function]"
+  [PASS] "gcr.io/kpt-fn/set-labels:v0.1.4[function]" in 0s
```

Comment:
I think we need to have an image with more than one executables and invoke them in sequence with --image-pull-policy=ifNotPresent. You pre-build an image and store it at gcr.io/kpt-fn-demo. It would be even great to test the time taken for execution.

Reply:
Done.